qa/L0_sdk/test.sh

```diff
@@ -34,19 +34,11 @@ set +e
 
 RET=0
 
-# Check image_client and perf_client
+# Check image_client
 if [[ ! -x "triton_client/bin/image_client" ]]; then
     echo -e "*** image_client executable not present\n"
     RET=1
 fi
-if [[ ! -x "triton_client/bin/perf_analyzer" ]]; then
```

Comment:
This validation need to be adjusted as `perf_analyzer should be installed.

Reply:
Sure. I modified the validation.